app/assets/javascripts/student_profile/IncidentHeatmap.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import DashboardScatterPlot from '../school_administrator_dashboard/DashboardScatterPlot';
+
+export default class IncidentHeatmap extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getincidentTimeAsMinutes(incident) {
```

Comment:
This works but `getincidentTimeAsMinutes` looks like it's copy-pasted from `SchoolDisciplineDashboard`.  Could you move it (and `areMinutesWithinSchoolHours`) to a plain function that `DashboardScatterPlot` exports?  That way both callers can use that same function, and maybe that helps with what you were asking about re: "setting the school hours in one location?"